observation_portal/sciapplications/serializers.py

```diff
@@ -46,14 +46,31 @@ class ScienceApplicationReviewSerializer(serializers.ModelSerializer):
     is_secondary_reviewer = serializers.SerializerMethodField()
     user_reviews = ScienceApplicationUserReviewNestedSerializer(read_only=True, many=True)
     my_review = serializers.SerializerMethodField()
+    long_term = serializers.SerializerMethodField()
+    total_requested_time_0m4 = serializers.SerializerMethodField()
+    total_requested_time_1m0 = serializers.SerializerMethodField()
+    total_requested_time_2m0 = serializers.SerializerMethodField()
 
     class Meta:
         model = ScienceApplicationReview
         fields = [
           "id", "science_category", "technical_review", "status", "mean_grade", "summary",
           "title", "semester", "abstract", "pdf_url", "completed", "can_summarize", "is_primary_reviewer", "is_secondary_reviewer", "user_reviews", "my_review",
+          "long_term", "total_requested_time_0m4", "total_requested_time_1m0", "total_requested_time_2m0",
         ]
 
+    def get_total_requested_time_0m4(self, obj):
```

Comment:
This is all specific to aperture sizes LCO uses... Since the obs portal is open source for other observatories as well I would prefer to keep code in the base obs portal generic for these types of things. Maybe instead you could make a serializer field `total_requested_time_by_aperture` which returns a dictionary keyed on aperture size with the value of the total time requested at that telescope class size?